app/src/main/AndroidManifest.xml

```diff
@@ -1,25 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="vn.fstyle.com.androidWheelPickerView">
+<manifest package="vn.fstyle.com.androidWheelPickerView">
 
-    <uses-permission android:name="android.permission.INTERNET" />
-
-    <application
-        android:name=".App"
-        android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
-        android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name="vn.fstyle.com.androidWheelPickerView.MainActivity">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
-                <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
-        </activity>
-        <meta-data
-            android:name="io.fabric.ApiKey"
-            android:value="afa5b4969ced7faced055aca28be45ea3adfaa5f" />
-    </application>
+    <application />
```

Comment:
WARNING: [MissingApplicationIcon] Should explicitly set `android:icon`, there is no default
 You should set an icon for the application as whole because there is no default. This attribute must be set as a reference to a drawable resource containing the image (for example `@drawable/icon`).